web-registry/src/components/organisms/RegistryLayout/RegistryLayout.config.tsx

```diff
@@ -22,17 +28,56 @@ export const getMenuItems = (pathname: string): HeaderMenuItem[] => [
         href: '/stats/activity',
         title: 'Activity',
         linkComponent: Link,
+        importCallback: (): Promise<any> => import('../../../pages/Activity'),
       },
       {
         pathname,
         href: '/ecocredit-batches/1',
         title: 'Ecocredit batches',
         linkComponent: Link,
+        importCallback: (): Promise<any> =>
+          import('../../../pages/EcocreditBatches'),
       },
     ],
   },
 ];
 
+interface GetUserMenuItemsParams {
+  pathname: string;
+  linkComponent: React.FC<NavLinkProps>;
+  theme: Theme;
+}
+
+export const getUserMenuItems = ({
+  linkComponent,
+  pathname,
+  theme,
+}: GetUserMenuItemsParams): HeaderDropdownItemProps[] =>
+  [
+    {
+      pathname,
+      linkComponent,
+      title: 'My Portfolio',
+      href: '/ecocredits/portfolio',
+      icon: (
+        <CreditsIcon
+          sx={{ height: 18, width: 20 }}
+          color={theme.palette.secondary.main}
+        />
+      ),
+      importCallback: (): Promise<any> => import('../../../pages/Dashboard'),
```

Comment:
Optional importCallback to preload js chunk on menu hover.

Reply:
interesting..